server/http/src/service.rs

```diff
@@ -219,7 +225,25 @@ where
                         .run_query(query)
                         .map_err(|e| GraphQLServerError::from(e))
                 })
-                .then(|result| GraphQLResponse::new(result)),
+                .then(move |result| {
+                    let elapsed = start.elapsed().as_millis();
+                    match result {
+                        Ok(_) => info!(
+                            logger,
+                            "GraphQL query served.";
```

Comment:
Another thing: I believe we're already logging query times in the `graph-graphql` crate. Those messages should probably be removed then.